freedomofpress/trackthenews
# Switch from setup.py to Poetry

To test:

Use the [readme](https://github.com/freedomofpress/trackthenews/blob/bfbaaa1aec780c128634d6ddd213dbff15b0df8b/README.md) to set up the project dev environment and confirm you can make it from cloning the project to getting a bot that will tweet or toot news stories.

Be attentive to following the process to set up the dev environment at the bottom of the readme, _not the instructions at the top_ which are for using the package as a library.

This PR is available to review for anyone on the web team
